ee/tabby-webserver/src/service/page/prompt_tools.rs

```diff
@@ -0,0 +1,130 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use tabby_inference::ChatCompletionStream;
+
+use crate::service::utils::prompt::{request_llm, transform_line_items};
+
+pub(super) async fn pipeline_page_title(
```

Comment:
no need to specify `(super)` as the mod itself is only visible to its parent (`mod prompt_tools`, instead of `pub mod prompt_tools`)